testing/models.test.js

```diff
@@ -41,11 +48,609 @@ afterAll(async () => {
+        const id = "635c4d9428ec6a0f12007709";
+        const reviews = await foodServices.getReviews(id);
+        expect(reviews.length).toBe(target);
+    });
+
+    test("get all the reviews for a given food id - error", async () => {
+        const id = "635c09";
+        const reviews = await foodServices.getReviews(id);
+        expect(reviews).toBeFalsy();
+    });
+
+    test("add 1 review - no error", async () => {
+        const id = "635c4d9428ec6a0f12577719";
+        const review = "amazingly delicious";
+        const result = await foodServices.addReview(id, review);
+        expect(result).toBeTruthy();
+        expect(result.review).toBe(review);
+        expect(result).toHaveProperty("food_id");
+    });
+
+    test("add 1 review - error", async () => {
+        const id = "34dw2";
+        const review = "amazingly delicious";
+        const result = await foodServices.addReview(id, review);
+        expect(result).toBeFalsy();
+    });
+});
+
+describe("tests incrementFoodRating", () => {
+    afterEach(async () => {
+        await foodModel.deleteMany();
+    });
+
+    test("increment likes", async () => {
+        let dummyFood = {
+            name: "agave",
+            ingredients: "organic blue agave nectar",
+            restaurant: "Balance Cafe",
+            vegetarian: true,
+            vegan: true,
+            nutrition: {
+                calories: 176,
+                fat: 0,
+                sodium: 2,
+                carbs: 43,
+                fiber: 0,
+                sugar: 1,
+                protein: 1,
+            },
+            likes: 50,
+            dislikes: 23,
+            poisonings: 14,
+            price: 1,
+            restaurant_id: "635b7be05455166813f787ad",
+        };
+        let foodResult = new foodModel(dummyFood);
+        await foodResult.save();
+
+        const id = foodResult.id;
+        const rating = "likes";
+        const result = await foodServices.incrementFoodRating(id, rating);
+        expect(result).toBeTruthy();
+    });
+
+    test("increment dislikes", async () => {
+        let dummyFood = {
+            name: "agave",
+            ingredients: "organic blue agave nectar",
+            restaurant: "Balance Cafe",
+            vegetarian: true,
+            vegan: true,
+            nutrition: {
+                calories: 176,
+                fat: 0,
+                sodium: 2,
+                carbs: 43,
+                fiber: 0,
+                sugar: 1,
+                protein: 1,
+            },
+            likes: 50,
+            dislikes: 23,
+            poisonings: 14,
+            price: 1,
+            restaurant_id: "635b7be05455166813f787ad",
+        };
+        let foodResult = new foodModel(dummyFood);
+        await foodResult.save();
+
+        const id = foodResult.id;
+        const rating = "dislikes";
+        const result = await foodServices.incrementFoodRating(id, rating);
+        expect(result).toBeTruthy();
```

Comment:
Maybe for these increment functions, could have the test being if the likes or dislikes were actually incremented? I'm just worried that he might have issue with this.